src/client/datascience/jupyter/debuggerVariables.ts

```diff
@@ -182,7 +183,7 @@ export class DebuggerVariables extends DebugLocationTracker
         end: number,
         kernel?: IKernel,
         sliceExpression?: string
-    ): Promise<{}> {
+    ): Promise<{ data: Record<string, unknown>[] }> {
```

Comment:
In our code, we know exactly what we're expecting (or what its supposed to look like), hence provided better typing
Else its difficult to tell what's going where